src/LagoonCommands.php

```diff
@@ -117,6 +116,72 @@ public function aliases() {
     }
   }
 
+  /**
+   * Get and print remote aliases from lagoon API site aliases file.
+   *
+   * @param string $file
+   *   Optional, output the alias file to a particular file.
+   *
+   * @command lagoon:generate-aliases
+   *
+   * @aliases lg
+   */
+  public function generateAliases($file = NULL) {
+    // Project still not defined, throw a warning.
+    if ($this->projectName === FALSE) {
+      $this->logger()
+        ->warning('ERROR: Could not discover project name, you should define it inside your .lagoon.yml file');
+      return;
+    }
+
+    if (empty($this->jwt_token)) {
+      $this->jwt_token = $this->getJwtToken();
+    }
+
+    $response = $this->getLagoonEnvs();
+    // Check if the query returned any data for the requested project.
+    if (empty($response->data->project->environments)) {
+      $this->logger()
+        ->warning("API request didn't return any environments for the given project '$this->projectName'.");
+      return;
+    }
+
+    foreach ($response->data->project->environments as $env) {
+      $details = [
+        "host" => $env->kubernetes->sshHost,
```

Comment:
we need to handle existing deploytargets here that have no specific ssh host set - otherwise they will return `null` as the host?

I'm just not sure where to read it from - we could fall back to the lagoon aliases (or a hard coded default?

Reply:
@tobybellwood - I think that should do it.